src/library/Uno.Themes.WinUI.Markup/Theme.LightWeight/Themes.LightWeightStyles.CheckBox.cs

```diff
@@ -0,0 +1,245 @@
+							public static ResourceValue<Brush> IndeterminatePressed => new("CheckBoxGlyphForegroundIndeterminatePressed", true);
+
+							[ResourceKeyDefinition(typeof(Brush), "CheckBoxGlyphForegroundIndeterminateDisabled")]
+							public static ResourceValue<Brush> IndeterminateDisabled => new("CheckBoxGlyphForegroundIndeterminateDisabled", true);
+						}
+
+						public static class PathData
+						{
+							[ResourceKeyDefinition(typeof(string), "CheckBoxHyphenGlyphPathData")]
+							public static ResourceValue<string> Hyphen => new("CheckBoxHyphenGlyphPathData", true);
+
+							[ResourceKeyDefinition(typeof(string), "CheckBoxCheckGlyphPathData")]
+							public static ResourceValue<string> Check => new("CheckBoxCheckGlyphPathData", true);
+						}
+					}
+
+					public static class Typo
+					{
+						[ResourceKeyDefinition(typeof(FontFamily), "CheckBoxFontFamily")]
+						public static ResourceValue<FontFamily> FontFamily => new("CheckBoxFontFamily", true);
+#if WinUI
+						[ResourceKeyDefinition(typeof(FontWeights), "CheckBoxFontWeight")]
+						public static ResourceValue<FontWeights> FontWeight => new("CheckBoxFontWeight", true);
+#else
+						[ResourceKeyDefinition(typeof(FontWeight), "CheckBoxFontWeight")]
+						public static ResourceValue<FontWeight> FontWeight => new("CheckBoxFontWeight", true);
+#endif
+						[ResourceKeyDefinition(typeof(double), "CheckBoxFontSize")]
+						public static ResourceValue<double> FontSize => new("CheckBoxFontSize", true);
+
+						[ResourceKeyDefinition(typeof(double), "CheckBoxCharacterSpacing")]
+						public static ResourceValue<double> CharacterSpacing => new("CheckBoxCharacterSpacing", true);
+					}
+
+					public static class FocusArea
+					{
+						public static class Size
+						{
+							[ResourceKeyDefinition(typeof(double), "CheckBoxFocusAreaSize")]
+							public static ResourceValue<double> Default => new("CheckBoxFocusAreaSize", true);
+						}
+					}
+
+					public static class CheckArea
+					{
+						public static class Size
+						{
+							[ResourceKeyDefinition(typeof(double), "CheckBoxCheckAreaSize")]
+							public static ResourceValue<double> Default => new("CheckBoxCheckAreaSize", true);
+						}
+
+						public static class Length
+						{
+							[ResourceKeyDefinition(typeof(GridLength), "CheckBoxCheckAreaLength")]
+							public static ResourceValue<GridLength> Default => new("CheckBoxCheckAreaLength", true);
+						}
+
+						public static class CornerRadiuses
+						{
+							[ResourceKeyDefinition(typeof(CornerRadius), "CheckBoxCheckAreaCornerRadius")]
+							public static ResourceValue<CornerRadius> Default => new("CheckBoxCheckAreaCornerRadius", true);
+						}
+
+						public static class BorderThickness
+						{
+							[ResourceKeyDefinition(typeof(Thickness), "CheckBoxCheckAreaBorderThickness")]
+							public static ResourceValue<Thickness> Default => new("CheckBoxCheckAreaBorderThickness", true);
+						}
```

Comment:
Missing CheckArea Padding @nguyeenvy to match the XAML
![image](https://github.com/unoplatform/Uno.Themes/assets/16295702/0b5cb335-2f22-40e0-b5f6-b46d9e0bdeed)